_posts/2022-mm-dd-crypto-1/2022-mm-dd-crypto1-history-and-basics.markdown

```diff
@@ -0,0 +1,157 @@
+* The age of modern computing, with its ever-increasing amount of computational power (quantum computing, anyone?) poses new problems and threats to proven approaches.
+
+"But where does he want to go with this?", you may want to ask. At this point, cryptography is something that concerns us all. It's in our phones, our banks, our computers and even in our refrigerators and ceiling lamps (hopefully). And I think it's important to understand what's going on. To a degree at least. And to understand where we stand and where we're going, we need to understand where we come from. If you take anything from this paragraph, let it be "the enemy knows the system" - never trust complexity as a security device.
+
+To answer the initial question posed in the alt title: cryptography is a means of ensuring that important information can only be accessed by people/parties you're comfortable sharing it with.
+
+### Notation/Ground rules
+Before we begin, we need to agree on a few things. The first of which is "how do we actually write this down?". To keep things consistent across two authors and a number of posts, we'll define the most important details right at the start.
+
+#### Names
+Examples in the realm of cryptography always rely on names to illustrate who is interacting with whom. The most commonly used names are:
+
+**Alice**
+Alice is our "person A". She either pens the original message that starts a (hopefully encrypted) conversation or is the one that states our general problem.
+
+**Bob**
+Similarly, Bob is "person B" and usually the one to interact with Alice.
+
+**Eve**
+Eve is the **e**vil attacker that tries to interject, manipulate, listen to, ... whatever it is Alice and Bob are discussing.
+
+**Oscar**
+Oscar is our **o**pponent and, similar to Eve, tries to interfere with the communication between Alice and Bob. He might occasionally show up for the sake of flavour.
+
+We may at times need to introduce additional parties. For the sake of simplicity, all evil attackers will use names that start with an E and all opponents will start with an O. All other actors will draw from the remainder of the alphabet.
+
+#### Notation
+When talking about cryptography, especially when doing to in theory, there is not really a way around using math one way or another. We will try to focus on "concept first". This means that we'll introduce math as necessary, explaining new symbols as they come into play instead of building a giant glossary in this post. 
+> Mathematical notation/definitions will be put in quotes.
+
+Also, we'll sometimes use examples to illustrate our points. If we do, we'll follow these guidelines:
+* `Cryptographic Information (plain/cipher texts, keys, ...) is represented by inline code`. This applies to entire sentences as well as single letters and words.
```

Comment:
Might be a bit cumbersome and I don't know if the LaTeX-implementation supports it, but I generally like putting cryptographic algorithms in `\textsc` and variables in `\mathsf`, giving a smallcaps resp. serif-free font.

I'd also prefer using LaTeX instead of code (using `\(` and `\)` for inline math), as it *will* get ugly for stuff like RSA (where you're having powers).